src/components/Auth/Login/Login.react.js

```diff
@@ -148,6 +148,27 @@ class Login extends Component {
     }
   };
 
+  checkAdmin = () => {
+    actions
```

Comment:
This should be `this.props.actions`. Please check the code before submitting.

Reply:
Actually actions is already defined, no need to use this.props.actions in this file